core/print_api.php

```diff
@@ -752,14 +752,15 @@ function print_profile_option_list_from_profiles( array $p_profiles, $p_select_i
  * @param integer $p_project_id  A project identifier.
  * @return void
  */
-function print_category_option_list( $p_category_id = 0, $p_project_id = null ) {
+function print_category_option_list( $p_category_id = 0, $p_project_id = null, $p_active_only = false ) {
```

Comment:
Update phpdoc config with new parameter.

Reply:
Still missing PHPDoc

Reply:
Which file needs to be updated with the parameter description?